dwellingly/src/views/addProperty.js

```diff
@@ -0,0 +1,162 @@
+                                                className="column form-field"
+                                                type="text"
+                                                name="city"
+                                                onChange={handleChange}
+                                                value={values.city}
+                                                placeholder="Portland"
+                                            />
+                                            {errors.city ? (<div className="error-message">{errors.city}</div>) : null}
+                                        </div>
+                                        <div className="form-row columns">
+                                            <label className="column is-one-fifth" htmlFor="state">State</label>
+                                            <Field
+                                                className="column form-field"
+                                                type="text"
+                                                name="state"
+                                                onChange={handleChange}
+                                                value={values.state}
+                                                placeholder="OR"
+                                            />
+                                            {errors.state ? (<div className="error-message">{errors.state}</div>) : null}
+                                        </div>
+                                        <div className="form-row columns">
+                                            <label className="column is-one-fifth" htmlFor="zipcode">Zipcode</label>
+                                            <Field
+                                                className="column form-field"
+                                                type="text"
+                                                name="zipcode"
+                                                onChange={handleChange}
+                                                value={values.zipcode}
+                                                placeholder="97217"
+                                            />
+                                            {errors.zipcode ? (<div className="error-message">{errors.zipcode}</div>) : null}
+                                        </div>
+                                        <div className="form-row columns">
+                                            <label className="column is-one-fifth" htmlFor="zipcode">Units</label>
+                                            <Field
+                                                className="column form-field"
+                                                type="text"
+                                                name="units"
+                                                onChange={handleChange}
+                                                value={values.zipcode}
```

Comment:
Looks like this line has a copy/paste issue.
I noticed when I type in the 'zip' form element, it adds my input to the "Unit" box, and I can't type into the Unit box directly.

![image](https://user-images.githubusercontent.com/8600792/78210602-8462bd00-745f-11ea-8c3e-0ac0122fa5fb.png)